src/en/contact/contact.md

```diff
@@ -1,12 +1,30 @@
 ---
-title: Contact us
+title: Get in contact
 layout: "layouts/base.njk"
 eleventyNavigation:
   key: contactusEN
   title: Contact us
   locale: en
   order: 3
 translationKey: "contactus"
+contactForm: en
 ---
 
-# Contact us
+# Get in contact
```

Comment:
I think we discussed in the meeting yesterday to change this heading to "Contact us"